SetUp/autoDefine/commandWriters.py

```diff
@@ -193,6 +193,35 @@ def assignSym(botSpecs, entries, defInp):
             else:
                defInp.write('sy '+group+'\n')
 
+#fix() specifies part of geometry to restrain
+#within internal coordinate sub-menu.
+#Only type=tors currently supported.
+#
+#Default: $fix omitted - restraint not used
+#
+#KeyFormat: $fix type=[label] atoms=[label1,label2,label3,label4]
```

Comment:
That's my shorthand for a list of integers, it's meant to show ranges and comma separated lists are allowed. Maybe not that clear without context.